site/astro.config.js

```diff
@@ -0,0 +1,121 @@
+// @ts-check
+import { defineConfig } from 'astro/config'
+import sitemap from '@astrojs/sitemap'
+import starlight from '@astrojs/starlight'
+
+// https://astro.build/config
+export default defineConfig({
+  site: 'http://cli.netlify.com',
+  trailingSlash: 'ignore',
+  integrations: [
+    sitemap({
+      lastmod: new Date(),
+    }),
+    starlight({
+      credits: true,
+      title: 'Netlify CLI command reference',
+      description: 'Full command reference for the Netlify CLI',
+      social: {
```

Comment:
TODO we could add more socials here

Reply:
added youtube, github, twitter, bluesky (also found instagram and linkedin but feels weird here)